pkgs/development/python-modules/baycomp/default.nix

```
@@ -0,0 +1,29 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, numpy
+, scipy
+, scikit-learn
+, matplotlib
+}:
+
+buildPythonPackage rec {
+  pname = "baycomp";
+  version = "1.0.2";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-xDRywWvXzfSITdTHPdMH5KPacJf+Scg81eiNdRQpI7A=";
+  };
+
+  propagatedBuildInputs = [ numpy scipy scikit-learn matplotlib ];
```

```suggestion
  propagatedBuildInputs = [
    numpy
    scipy
    matplotlib
  ];
```
It seems this package doesn't need scikit-learn.
https://github.com/janezd/baycomp/blob/9fc56da7c8b56eb1dbd2df725e18606818434a20/setup.py#L23-L29